### ISSUE TITLE
Cannot mount bucket with riofs

### ISSUE THREAD

**Issue by wawrzek** [skoobe/riofs#93]
I cannot mount bucket with riofs desptite proper credentials.

riofs fails:
root@ip:~/riofs-master/src# ./riofs mybucket /s3/bucket
ERROR! [con: 0x1856cd0] Server returned HTTP error: 403 (Forbidden)! Retry ID: 1 of 20
[...]
ERROR! [con: 0x1856cd0] Server returned HTTP error: 403 (Forbidden)! Retry ID: 20 of 20
ERROR! [con: 0x1856cd0] Reached the maximum number of retries !
ERROR! Failed to execute HTTP request !
ERROR! Failed to get bucket ACL! Most likely you provided wrong AWS keys or bucket name !

even if aws cli works fine.
root@ip:~/riofs-master/src#  aws s3 ls s3://mybucket
2015-02-10 14:58:02   25030165 Terminology 0.3.webm
2015-02-10 13:24:46        110 credentials.csv
2015-02-10 14:25:47          0 test

What might be the problem.

**Comment by wizzard** [comment]
Hello,
as it says, most likely you specified the wrong credentials or bucket name. 
As "bucket name" try to specify a bucket name as one word (i.e. "bucket", instead of "/s3/bucket").

Hope it helps you!

**Comment by wawrzek** [comment]
@wizzard - What do you mean? Based on riofs help:
    ubuntu@ip241:~$ riofs --help
    Usage:
      riofs [OPTION...] [bucketname] [mountpoint]

I do everything fine, because **/s3/bucket** (second argument) is only a mountpoint.

**Comment by jardimsonoro** [comment]
I am facing the same issue (403) with DreamObjects

**Comment by jackyq2015** [comment]
same issue. I keep getting this error message even the bucket was mounted properly(I can list/read/write the files). when I do a ps, I can see the /usr/local/bin/riofs process still there.

**Comment by skinniyah** [comment]
This is probably because you're trying to access a bucket in a newer (post 2014) region like Frankfurt. Newer regions use v4 signing and riofs currently supports only v2 signing. Haven't yet found documentation on how to fix this, but am looking. Hope this issue is resolved soon!

**Comment by wizzard** [comment]
Correct, currently RioFS does not support all new regions (Frankfurt, Asia region). 
Please refer to issue #85, I'll update that ticket when I implement the support.

**Comment by jackyq2015** [comment]
Actually, I think I am in the US-east region. Also when I tried to check s3 region from the menu of right corner, it prompt that "S3 does not require region selection.".

**Comment by siddharthmakwana** [comment]
Does riofs even support https protocol ??

**Comment by wizzard** [comment]
@siddharthmakwana yes, it does. It requires ```libevent-openssl``` package (or similar).

**Comment by jadbox** [comment]
Everything was working fine for me the past few weeks... today I'm getting 403 errors for my buckets (all in US regions).

**Comment by markl17** [comment]
same here used to work then I tried mounting another one it gave me tha error so I killed the process and now I cant even get the original to work

**Comment by audiofeature** [comment]
Any update on this? Is the v4 signing now supported, or anytime soon? Otherwise riofs can't be used by many.

**Comment by gvdr** [comment]
One year later, still same error. Anything new on this front?
